### DISCUSSION TITLE
Search by lat and lng

### DISCUSSION THREAD

**Discussion by wilsondossantos** [directus/directus#17766] [Archive]
Hi there, 

My Directus project has a collection with restaurants. Each restaurant currently has a field with lat and lng coordinates in a Point type field. 

Is it possible to use Directus API to fetch restaurants nearby a lat and lng coordinates?

**Comment by Zack-Heisnberg** [comment]


**Reply by wilsondossantos** [reply]
Thanks Zack. Do you have an exemple of how to use _intersect to filter records based on a distance from a lat/lng?

**Comment by hennroja** [comment]
https://github.com/directus/directus/discussions/25123
Related
